standards/alerting.md

```diff
@@ -1,3 +1,45 @@
-# Alerting & Notifications
+# Monitoring & Alarms
 
-This is a placeholder for linking until the standard has been drafted.
\ No newline at end of file
+All components MUST have an alarm set up.
```

Comment:
Good point. Honestly, I wasn't sure what to word to use. A "service" to me is something that serves - generally through an API interface like HTTP, etc. I guess I've been using the word "component" to mean service, consumer, etc. But, you're right, it's not a great word. Maybe I'll just change it to "services, consumers, and applications".